syncd/SelectableEventHandler.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "swss/logger.h"
+#include "swss/sal.h"
+
+namespace syncd
+{
+
+    // This class implements handler for Selectable events.
+    class SelectableEventHandler {
+        public:
+
+            virtual ~SelectableEventHandler()
+            {
+              SWSS_LOG_ENTER();
+            }
```

Comment:
if its not a template, move code to cpp file

Reply:
Do we see any issue with keeping the destructor here in .h file?

Reply:
Usually when code is not template we prefer to move cpp code to cpp file, and also add empty intentionally or = default